libs/ngx-dsfr/stepper/src/lib/stepper.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="fr-stepper">
+  <h2 class="fr-stepper__title">
+    <span class="fr-stepper__state">{{stepSubtitle}}</span>
```

Comment:
The way I see it, we don't need the `stepSubtitle` property. By reading the documentation, it seems like this part of the Component must be restrained in terms of what it looks like.
As we already have what we need for that, I suggest to remove the `stepSubtitle` property in the Component and replace this line with :
```html
<span class="fr-stepper__state">Étape {{stepNumber}} sur {{numberOfSteps}}</span>
```